src/lib/http/index.ts

```diff
@@ -77,18 +78,21 @@ export function PUT<T>(requestURL: string, data: Params = {}, { headers = {} }:
   });
 }
 
-function _getHttpConfig(data: Params | null | undefined | string, config?: HttpConfig): HttpConfig {
-  // allow passing config as second or 3rd argument
-  const httpConfig = config || _isConfigObject(data) ? (data as HttpConfig) : {};
```

Comment:
FYI, I think there was a bug in this line, which required type casting:
The JIT parses this as `(config || _isConfigObject(data)) ? data : {}` by default, so that if config is defined, we end up using data. I was able to remove the type-casting when I rewrote it this way:
`config || (isConfigObject(data) ? data : {})`.